Haskell/MaîtreD.hs

```diff
@@ -1,11 +1,19 @@
 module MaîtreD where
 
-import Data.Time (ZonedTime)
+import Data.Time (ZonedTime (zonedTimeToLocalTime, zonedTimeZone))
```

Comment:
>but what does it accomplish to explicitly import them?

Currently, in this sample module, we need only `zonedTimeToLocalTime` and `zonedTimeZone`, all in a single place.

Why would you prefer `ZonedTime(..)` instead?